ui/src/App.tsx

```diff
@@ -2,11 +2,17 @@ import { useState } from 'react';
 import {
   AppBar,
   Box,
+  // Button,
   Container,
+  // Dialog,
+  // DialogActions,
+  // DialogContent,
+  // DialogTitle,
   IconButton,
   Stack,
   Tab,
   Tabs,
+  // TextField,
```

Comment:
```suggestion
  Container,
  IconButton,
  Stack,
  Tab,
  Tabs,
```

Always remove unused imports